### DISCUSSION TITLE
Will this repository become a PublicArchive?

### DISCUSSION THREAD

**Discussion by MizuiMiduki** [squizlabs/PHP_CodeSniffer#3955] [Q&A] [answered]
The README says that this repository is no longer in development, but are there plans to fully archive the repository itself or make it a public archive?
If possible, I would appreciate it if you could make it a Public Archive to prevent future confusion about PHPCS information.

(This is entirely my personal opinion, so please take it with a pinch of salt, but I believe it is the responsibility of OSS developers to properly archive their repositories after they finish development.)

thank you

**Comment by jrfnl** [comment, marked as answer]
@MizuiMiduki The package is still in active development, just no longer in this repo, but in the https://github.com/PHPCSStandards/PHP_CodeSniffer repo.

As for archiving this repo, this will probably happen at some point, but wasn't done yet as archiving a repo locks all issues/PRs (makes them read-only). This would mean that no comments can be left anymore to ask people to transfer issues/PRs to the new repo, nor can people be informed when issues originally reported in this repo have been fixed in the new repo.

Along the same lines, I wouldn't have been able to reply to this discussion and inform you of the new repo, nor would you have been able to open this discussion.

**Reply by MizuiMiduki** [reply]
I see. I understand the situation.
When researching PHPCS on the Internet, you often find information about the new repository mixed up with information about the old repository.
It's a difficult problem, but I'd like to do something about it...
